### PR TITLE
Bread cape for bagel

### PR DESCRIPTION
**Pull request by humboldt123**
coool

### CODE REVIEW

**Review by 5HT2 — COMMENTED** [review]
Include the donators.json as well

**Comment by humboldt123** [comment]
Done

**Review by 5HT2 — APPROVED** [review]

**Comment by 5HT2** [comment]
tfw merge conflicts

**Comment by humboldt123** [comment]
What am i supposed to do

**Comment by humboldt123** [comment]
fiexe

**Comment by 5HT2** [comment]
make a new pull based off of the current repo or fix the conflicts

File: assets/donators.json [hunk 1]
```diff
@@ -375,8 +375,12 @@
...
     "type": "-1"
   },
+  {
+    "uuid": "f14c348d-4e38-415a-aba5-8688c71d7288",
+    "type": "-1"
+  },
   {
     "uuid": "75e17a15-c1bc-4167-bda4-e7a63be0f465",
     "type": "-1"
-  }
```
Comment (5HT2) [inline assets/donators.json:381]: wth is this

Comment (humboldt123) [reply assets/donators.json:381]: idk

Comment (humboldt123) [reply assets/donators.json:381]: image

**Comment by humboldt123** [comment]
I just fixed the conflicts

**Comment by 5HT2** [comment]
I'll take a look later

**Comment by 5HT2** [comment]
no bread cape allowed :rage:

**Comment by FluorideHistorian** [comment]
give him bread cape and give me eclipse cape

**Comment by FluorideHistorian** [comment]
and give xiaro egg cape 😳

**Comment by humboldt123** [comment]
:trollface:

**Pull request closed without merging**